internal/pkg/fieldtags/testdata/src/tests/test.go

```diff
@@ -15,12 +15,11 @@
 package fieldtags
 
 type Person struct {
-	password, creds          string      `levee:"source"`               // want "tagged field: password, creds"
-	secret                   string      `json:"secret" levee:"source"` // want "tagged field: secret"
-	another                  interface{} "levee:\"source\""             // want "tagged field: another"
```

Comment:
This PR yields no longer supporting these escaped-quote style of tags.  We could jump through some hoops to do so if necessary, but if the point was to keep the lookup simple, I'd rather wait until someone complains that try to preempt the issue.